x-pack/plugins/cases/README.md

```diff
@@ -41,43 +41,38 @@ cases: CasesUiStart;
     onSuccess,
   })
 ```
+
 ##### Methods:
 ### `getAllCases`
 Arguments: 
 
 |Property|Description|
 |---|---|
-|configureCasesHref|`string;` route for configure cases page
-|createCaseHref|`string;` route for create cases page
+|caseDetailsNavigation|`{ getHref: (caseDetails: CaseDetailsHrefSchema) => string; onClick: (caseDetails: CaseDetailsHrefSchema) => void }` route configuration to generate the case details url for the case details page
+|configureCasesNavigation|`{ href: string; onClick: (arg: React.MouseEvent | MouseEvent) => void }` route configuration for configure cases page
```

Comment:
nit: Something is missing and it renders as 

<img width="620" alt="Screenshot 2021-04-08 at 11 35 42 AM" src="https://user-images.githubusercontent.com/7871006/113995116-9f2ff480-985e-11eb-8bdf-d2e9aa226d19.png">

Reply:
yeah it doesnt like the `|` because thats part of the table schema. check out how I did `onRowClick`, I cheated and used `!` since it looks like `|`. If someone has a better solution please update it there as well

Reply:
You can escape the `|` by going `\|`

Reply:
Thanks for the tip. Done!